README

```diff
@@ -40,7 +40,7 @@ Requirements:
 
 Optional dependencies:
 
-  * cld2 (better language autodetection and non-English source languages)
```

Comment:
This change is semantically incorrect — it really is *and*, not *in*.

Reply:
Ok, I was unsure about this one (did not understand the meaning). I will update